server/database.go

```diff
@@ -196,6 +196,33 @@ func tryChangePlayerUsername(senderUuid string, recipientUuid string, newUsernam
 	return nil
 }
 
+func handleTempBan(uuid string) (duration int, err error) {
+	var userExists int
+	db.QueryRow("SELECT EXISTS (SELECT * FROM accounts WHERE uuid = ?)", uuid).Scan(&userExists)
+
+	if userExists == 0 {
+		return "", errors.New("user not found")
+	}
+
+	err = tryBanPlayer(uuid, uuid)
+	
+	if err != nil {
+		handleInternalError(http.ResponseWriter, *http.Request, err)
+		return
+	}
+	
+	tempBan := time.NewTimer(duration * 60 * time.Second)
```

Comment:
This will cause the HTTP responder to hang without giving a response. You might want to set up a timer to be called later instead, push notifications are already doing this so you can refer to it.